From 7ef2a8fe59a4b962546e0a6bba83a3097103df01 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Tue, 10 Jun 2008 21:58:35 -0400
Subject: [PATCH 1/2] arm: serialize build targets

The image targets need to trigger after we've build all their
dependencies. To do this we add .NOTPARALLEL and  ensure that
the image targets are not built in parallel.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 arch/arm/Makefile |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/Makefile b/arch/arm/Makefile
index 356d702..b01504d 100644
--- a/arch/arm/Makefile
+++ b/arch/arm/Makefile
@@ -248,6 +248,9 @@ archprepare:
 # Convert bzImage to zImage
 bzImage: zImage
 
+# These targets cannot be built in parallel
+.NOTPARALLEL: zImage Image xipImage bootpImage uImage
+
 zImage Image xipImage bootpImage uImage: vmlinux
 	$(Q)$(MAKE) $(build)=$(boot) MACHINE=$(MACHINE) $(boot)/$@
 
-- 
1.6.5.2

